index.html

```diff
@@ -14,6 +16,10 @@
                     <li><a href="#about">About Me</a></li>
                     <li><a href="#projects">Projects</a></li>
                     <li><a href="#contact">Contact</a></li>
+                    <!-- added the skills and FAQ section -->
+                    <li><a href="#Skills">Skills</a></li>
```

Comment:
Let's maintain consistency. Could we keep the first letter of the ID name in lowercase, as in "#skills" and not "#Skills"? And the same goes for others too.